tests/test_preproc.py

```diff
@@ -1721,7 +1722,21 @@ def test_sync_ts_data_timestamps(self):
         dt = sync_timestamp[1] - sync_timestamp[0]
         num_padding = np.arange(sync_timestamp[0]-dt,0,-dt).shape[0]
         self.assertTrue(sync_data.shape[0], num_padding + sync_timestamp.shape[0])  
-              
+    
+    def test_destripe_lfp_batch(self):
```

Comment:
I think we also need a test that the function is grabbing the batches correctly. Maybe just make sure that each value in the destriped array is different from the value in the original array?

Reply:
I checked how much the destriped array was different from the original array. I found values of 1% elements changed from the original array. Almost all of elements in the destriped array was the same as the original elements. I'm not sure what test is appropriate for this function.

Reply:
Hmm I guess ideally we'd have a before and after image of neuropixel data that has been destriped. I'm not sure if that is necessary now, but could you add a link to the ibl white paper in the doc string for the function? (https://doi.org/10.6084/m9.figshare.19705522.v4)